File: database/src/main/java/com/example/ormlitetest/contentprovider/SecondActivity.java
package com.example.ormlitetest.contentprovider;

import android.database.Cursor;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import com.example.ormlitetest.R;

import java.util.ArrayList;
import java.util.List;

public class SecondActivity extends AppCompatActivity {

    private ListView listView;
    private ArrayAdapter<String> adapter;
    private List<String> contactList;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        contactList = new ArrayList<>();
        listView = (ListView) findViewById(R.id.listView);


    }

    private void readContacts() {
        Cursor cursor = null;
        try {
            cursor=getContentResolver().query(Con)
        } catch (Exception e) {

        }
    }


}

File: app/src/main/java/com/example/zhangqian/myapplication/activity/SecondActivity.kt
package com.example.zhangqian.myapplication.activity

import android.app.Activity
import android.os.Bundle
import com.example.zhangqian.myapplication.R
import kotlinx.android.synthetic.main.activity_second.*

class SecondActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)

        textView_second.text = "哈哈哈哈";
        button.text="设置新按钮名称啦"
        button.textSize=23f
    }


}
